misc/Kconfig

```diff
@@ -273,6 +273,25 @@ config BOOTLOADER_SRAM_SIZE
 	  This option specifies the amount of SRAM (measure in kB) reserved for
 	  when Zephyr is to act as a bootloader.
 
+config IS_MCUBOOT
+	bool
+	prompt "Act as an MCUboot bootloader"
+	default n
+	depends on XIP
+	depends on ARM
+	depends on IS_BOOTLOADER
+	select FLASH_PAGE_LAYOUT if FLASH_HAS_PAGE_LAYOUT
+	help
+	  This option indicates that Zephyr is being built as the MCUboot
+	  open source bootloader.
+
+config BOOTLOADER_MCUBOOT
```

Comment:
where is this being used?

Reply:
Nowhere for now honestly

Reply:
@nashif Now it is used in `TEXT_SECTION_OFFSET`